### PR TITLE
git-graph: init at unstable-2023-01-01

### PR DESCRIPTION
**Pull request by cafkafk**
###### Description of changes
Added git-graph package to nixpkgs

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by cafkafk** [comment]
@ofborg eval

**Review by emilylange — CHANGES_REQUESTED** [review]
Looks good so far!

Just two minor things :)

File: maintainers/maintainer-list.nix
[the remark this answers is not in the document]
Comment (cafkafk) [reply maintainers/maintainer-list.nix:1]: yea sure! didn't know what to call it

File: pkgs/applications/version-management/git-graph/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    repo = pname;
+    rev = "9bd54eb0aed6f108364bce9ad0bdff12077038fc";
+    hash = "sha256-tMM/mpt9yzZYSpnOGBuGLM0XTJiiyChfUrERMuyn3mQ=";
+  };
+
+  cargoHash = "sha256-ZLF/l2HnIbmkayWXhjYr01M6lGaGiK2UYyp654ncxgo=";
+
+  meta = with lib; {
+    description = "Command line tool to show clear git graphs arranged for your branching model";
+    homepage = "https://github.com/mlang-42/git-graph";
```
Comment (emilylange) [inline pkgs/applications/version-management/git-graph/default.nix:22]: Typo

```suggestion
    homepage = "https://github.com/mlange-42/git-graph";
```

**Review by emilylange — CHANGES_REQUESTED** [review]

**Review by emilylange — APPROVED** [review]

**Pull request merged**